src/backend/tests/models/charge_types/test_level_800_traffic_crime.py

```diff
@@ -125,11 +131,12 @@ def test_violation_level_charge_skips_analysis(self):
         assert charge.skip_analysis() is True
 
     def test_level_800_infractions_skip_analysis(self):
-        charge = ChargeFactory.create(statute='811135', level='Infraction Class B', disposition=self.convicted)
+        charge = ChargeFactory.create(statute='811.135', level='Infraction Class B', disposition=self.convicted)
 
         assert charge.skip_analysis() is True
 
 
+
```

Comment:
We forgot to change the statute for test cases below (i.e. skip analysis). As you mentioned while we were pair programming, the meaning of these tests have now changed if we don't switch away from chapter 813.